editor/src/clj/editor/material.clj

```diff
@@ -417,7 +398,7 @@
                 :type-unsigned-short num/normalized->ushort-double
                 :type-int num/normalized->int-double
                 :type-unsigned-int num/normalized->uint-double))]
-        (update new-attribute :values #(into (empty %) (map coerce-fn) %)))
+        (update new-attribute :values coll/transform (map coerce-fn)))
```

Comment:
`coll/transform` is like `coll/transfer`, but transfers into an empty collection of the same type as the input collection.